### PR TITLE
Add Blexin

### PR DESCRIPTION
**Pull request by turibbio**

### CODE REVIEW

**Review by edoardocostantinidev — CHANGES_REQUESTED** [review]
Hi @turibbio, career page url should point to a valid job list, can you please provide one?

**Comment by turibbio** [comment]
@edoardocostantinidev, we don't have the jobs page on our website. Should the LinkedIn page work?

**Comment by edoardocostantinidev** [comment]
@turibbio yes linkedin is fine, thanks.

File: data/cloud_software/blexin.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Blexin",
   "url": "https://www.blexin.com/",
-  "career_page_url": "https://www.blexin.com/",
+  "career_page_url": "https://www.linkedin.com/jobs/view/2945582736/",
```
Comment (edoardocostantinidev) [inline data/cloud_software/blexin.json:4]:
```suggestion
  "career_page_url": "https://www.linkedin.com/company/blexin/jobs/",
```

**Review by edoardocostantinidev — CHANGES_REQUESTED** [review]

File: data/cloud_software/blexin.json (hunk 1 above)
Comment (turibbio) [reply data/cloud_software/blexin.json:4]: Ok for me

**Review by edoardocostantinidev — APPROVED** [review]

**Pull request merged**